pywb/utils/io.py

```diff
@@ -1,12 +1,39 @@
 import zlib
 from contextlib import closing, contextmanager
+from tempfile import SpooledTemporaryFile
 
-from warcio.utils import BUFF_SIZE
 from warcio.limitreader import LimitReader
-from tempfile import SpooledTemporaryFile
+from warcio.utils import BUFF_SIZE
+
 
+def no_except_close(closable, raw=False):
```

Comment:
This can just be as we can just pass the raw object in, and ensure we release_conn if available:
```
def no_except_close(closable):
    if not closeable:
        return

    try:
       closeable.close()
    except:
       pass

    try:
        release_conn = getattr(closeable, 'release_conn', None)
        if release_conn is not None:
            release_conn()
    except:
       pass
```

Reply:
changes applied